package.json

```diff
@@ -4,7 +4,10 @@
   "description": "Selenium project",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "protractor protractor_testing/conf/conf.js && mocha",
+    "start": "mocha",
```

Comment:
Please don't name the testing script with the `start` alias.

Reply:
renamed as 'protractor' and 'mocha'